CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0]
+### Added
+* Multi-entrypoint capabilities to plugin
```

Comment:
```suggestion
* It's now possible to register multiple [HyP3 entry points (workflows) and run them through the main hyp3_isce2 entry point. 
```
Maybe something like this that's a little more descriptive. We should probably add a "how to" for adding a new hyp3 entry point, either in the README or `docs`.